network/80-6rd-tunnel.link

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT-0
+#
+# This config file is installed as part of systemd.
+# It may be freely copied and edited (following the MIT No Attribution license).
+#
+# To make local modifications, one of the following methods may be used:
+# 1. add a drop-in file that extends this file by creating the
+#    /etc/systemd/network/80-6rd-tunnel.link.d/ directory and creating a
```

Comment:
We should probably substitute these with `networkctl edit` :)

Reply:
I agree, but maybe let's do that in a later PR...

Reply:
> I agree, but maybe let's do that in a later PR...

Then, feel free to upgrade the green label.